### ISSUE TITLE
borg mount crashes with empty archive

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#1125]
if one tries to mount a completely empty archive, borg mount crashes.

note: usually this never happens as one does not have empty archives.

**Comment by ThomasWaldmann** [comment]
```
(borg-env)tw@tux:~/w/borg$ borg mount -f repo::1 mnt
Local Exception.
Traceback (most recent call last):
  File "/home/tw/w/borg/borg/fuse.py", line 173, in get_item
    return self.items[inode]
KeyError: 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/tw/w/borg/borg/archiver.py", line 1615, in main
    exit_code = archiver.run(args)
  File "/home/tw/w/borg/borg/archiver.py", line 1552, in run
    return args.func(args)
  File "/home/tw/w/borg/borg/archiver.py", line 85, in wrapper
    return method(self, args, repository=repository, **kwargs)
  File "/home/tw/w/borg/borg/archiver.py", line 488, in do_mount
    operations.mount(args.mountpoint, args.options, args.foreground)
  File "/home/tw/w/borg/borg/fuse.py", line 100, in mount
    signal = fuse_main()
  File "/home/tw/w/borg/borg/fuse.py", line 25, in fuse_main
    return llfuse.main(workers=1)
  File "src/fuse_api.pxi", line 343, in llfuse.main (src/llfuse.c:35968)
  File "src/handlers.pxi", line 80, in llfuse.fuse_getattr (src/llfuse.c:8699)
  File "src/handlers.pxi", line 81, in llfuse.fuse_getattr (src/llfuse.c:8646)
  File "/home/tw/w/borg/borg/fuse.py", line 185, in getattr
    item = self.get_item(inode)
  File "/home/tw/w/borg/borg/fuse.py", line 175, in get_item
    return self.cache.get(inode)
  File "/home/tw/w/borg/borg/fuse.py", line 43, in get
    self.fd.seek(inode - self.offset, io.SEEK_SET)
OSError: [Errno 22] Invalid argument

Platform: Linux tux 4.6.4-tw2 #4 SMP Mon Jul 18 15:44:28 CEST 2016 x86_64 x86_64
Linux: Ubuntu 16.04 xenial
Borg: 1.0.7.dev16+ng10ecb17  Python: CPython 3.5.1+
PID: 25973  CWD: /home/tw/w/borg
sys.argv: ['/home/tw/w/borg-env/bin/borg', 'mount', '-f', 'repo::1', 'mnt']
SSH_ORIGINAL_COMMAND: None
```

**Comment by PlasmaPower** [comment]
Do we just want to abort early if the archive is empty?

**Comment by enkore** [comment]
Empty directory would be better I think.

**Comment by PlasmaPower** [comment]
@enkore Makes sense, especially if this is used in scripts.

**Comment by ThomasWaldmann** [comment]
fixed by #1403.
